core/src/main/java/com/netflix/conductor/service/MetricService.java

```diff
@@ -131,6 +131,24 @@ public void taskRateLimited(String taskType, String refName, String defName) {
 		statsd.incrementCounter(aspect, toArray(tags));
 	}
 
+	public void httpStarted(String taskType, String refName, String defName, String service) {
```

Comment:
this Set creation was duplicated 6 times (only difference is the first item of the Set). I would use a common method for Set creation